tiledb/sm/c_api/tiledb.cc

```diff
@@ -5347,7 +5355,7 @@ const char* tiledb_timestamps() noexcept {
 /*            VERSION             */
 /* ****************************** */
 
-void tiledb_version(int32_t* major, int32_t* minor, int32_t* rev) noexcept {
+CAPI_INTERFACE_VOID(version, int32_t* major, int32_t* minor, int32_t* rev) {
```

Comment:
This function can't be wrapped because its not actually calling anything.

Reply:
Err, not sure if this makes sense in hindsight. Attempting to wrap this function fails because there's no corresponding `tiledb::api::tiledb_version` function because it's not using the exception wrapper.